packages/active-campaign-client/src/__tests__/handlers/addContact.test.ts

```diff
@@ -0,0 +1,32 @@
+import { handler } from '../../handlers/addContact';
+
+describe('addContact handler', () => {
+  it('should create a contact successfully', async () => {
+    const event = {
+      headers: {
+        Authorization: 'test-token',
+      },
+      body: JSON.stringify({
+        username: `test@example${new Date().getTime()}e.com`,
+        firstName: 'John',
+        lastName: 'Doe',
+        company: 'Test Co',
+        role: 'Developer',
+        mailinglistAccepted: true,
+      }),
+    };
+
+    const response = await handler(event as any);
+    expect(response.statusCode).toBe(200);
+  });
```

Comment:
Even adding env this test fail returning this error:
`

  console.error
    Error: AxiosError {
      port: 80,
      address: '127.0.0.1',
      syscall: 'connect',
      code: 'ECONNREFUSED',
      errno: -111,
      message: 'connect ECONNREFUSED 127.0.0.1:80',
      name: 'Error',
      config: {
        transitional: {
          silentJSONParsing: true,
          forcedJSONParsing: true,
          clarifyTimeoutError: false
        },
        adapter: [ 'xhr', 'http' ],
        transformRequest: [ [Function: transformRequest] ],
        transformResponse: [ [Function: transformResponse] ],
        timeout: 0,
        xsrfCookieName: 'XSRF-TOKEN',
        xsrfHeaderName: 'X-XSRF-TOKEN',
        maxContentLength: -1,
        maxBodyLength: -1,
        env: { FormData: [Function], Blob: [class Blob] },
        validateStatus: [Function: validateStatus],
        headers: Object [AxiosHeaders] {
          Accept: 'application/json, text/plain, */*',
          'Content-Type': 'application/json',
          'Api-Token': undefined,
          'User-Agent': 'axios/1.6.0',
          'Content-Length': '181',
          'Accept-Encoding': 'gzip, compress, deflate, br'
        },
        method: 'post',
        url: 'undefined/api/3/contacts',
        data: '{"contact":{"email":"test@example1730821628795e.com","firstName":"John","lastName":"Doe","fieldValues":[{"field":"company","value":"Test Co"},{"field":"role","value":"Developer"}]}}'
      },
      request: <ref *1> Writable {
        _writableState: WritableState {
          objectMode: false,
          highWaterMark: 16384,
          finalCalled: false,
          needDrain: false,
          ending: false,
          ended: false,
          finished: false,
          destroyed: false,
          decodeStrings: true,
          defaultEncoding: 'utf8',
          length: 0,
          writing: false,
          corked: 0,
          sync: true,
          bufferProcessing: false,
          onwrite: [Function: bound onwrite],
          writecb: null,
          writelen: 0,
          afterWriteTickInfo: null,
          buffered: [],
          bufferedIndex: 0,
          allBuffers: true,
          allNoop: true,
          pendingcb: 0,
          constructed: true,
          prefinished: false,
          errorEmitted: false,
          emitClose: true,
          autoDestroy: true,
          errored: null,
          closed: false,
          closeEmitted: false,
          [Symbol(kOnFinished)]: []
        },
        _events: [Object: null prototype] {
          response: [Function: handleResponse],
          error: [Function: handleRequestError],
          socket: [Function: handleRequestSocket]
        },
        _eventsCount: 3,
        _maxListeners: undefined,
        _options: {
          maxRedirects: 21,
          maxBodyLength: Infinity,
          protocol: 'http:',
          path: '/undefined/api/3/contacts',
          method: 'POST',
          headers: [Object: null prototype],
          agents: [Object],
          auth: undefined,
          family: undefined,
          beforeRedirect: [Function: dispatchBeforeRedirect],
          beforeRedirects: [Object],
          hostname: 'localhost',
          port: '',
          agent: undefined,
          nativeProtocols: [Object],
          pathname: '/undefined/api/3/contacts'
        },
        _ended: false,
        _ending: true,
        _redirectCount: 0,
        _redirects: [],
        _requestBodyLength: 181,
        _requestBodyBuffers: [ [Object] ],
        _onNativeResponse: [Function (anonymous)],
        _currentRequest: ClientRequest {
          _events: [Object: null prototype],
          _eventsCount: 7,
          _maxListeners: undefined,
          outputData: [],
          outputSize: 0,
          writable: true,
          destroyed: false,
          _last: true,
          chunkedEncoding: false,
          shouldKeepAlive: false,
          maxRequestsOnConnectionReached: false,
          _defaultKeepAlive: true,
          useChunkedEncodingByDefault: true,
          sendDate: false,
          _removedConnection: false,
          _removedContLen: false,
          _removedTE: false,
          strictContentLength: false,
          _contentLength: '181',
          _hasBody: true,
          _trailer: '',
          finished: false,
          _headerSent: true,
          _closed: false,
          socket: [Socket],
          _header: 'POST /undefined/api/3/contacts HTTP/1.1\r\n' +
            'Accept: application/json, text/plain, */*\r\n' +
            'Content-Type: application/json\r\n' +
            'User-Agent: axios/1.6.0\r\n' +
            'Content-Length: 181\r\n' +
            'Accept-Encoding: gzip, compress, deflate, br\r\n' +
            'Host: localhost\r\n' +
            'Connection: close\r\n' +
            '\r\n',
          _keepAliveTimeout: 0,
          _onPendingData: [Function: nop],
          agent: [Agent],
          socketPath: undefined,
          method: 'POST',
          maxHeaderSize: undefined,
          insecureHTTPParser: undefined,
          joinDuplicateHeaders: undefined,
          path: '/undefined/api/3/contacts',
          _ended: false,
          res: null,
          aborted: false,
          timeoutCb: null,
          upgradeOrConnect: false,
          parser: null,
          maxHeadersCount: null,
          reusedSocket: false,
          host: 'localhost',
          protocol: 'http:',
          _redirectable: [Circular *1],
          [Symbol(kCapture)]: false,
          [Symbol(kBytesWritten)]: 0,
          [Symbol(kNeedDrain)]: false,
          [Symbol(corked)]: 0,
          [Symbol(kOutHeaders)]: [Object: null prototype],
          [Symbol(errored)]: null,
          [Symbol(kHighWaterMark)]: 16384,
          [Symbol(kRejectNonStandardBodyWrites)]: false,
          [Symbol(kUniqueHeaders)]: null
        },
        _currentUrl: 'http://localhost/undefined/api/3/contacts',
        [Symbol(kCapture)]: false
      },
      cause: Error: connect ECONNREFUSED 127.0.0.1:80
          at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1495:16) {
        errno: -111,
        code: 'ECONNREFUSED',
        syscall: 'connect',
        address: '127.0.0.1',
        port: 80
      }
    }

      49 |     };
      50 |   } catch (error) {
    > 51 |     console.error('Error:', error);
         |             ^
      52 |     return {
      53 |       statusCode: 500,
      54 |       body: JSON.stringify({ message: 'Internal server error' }),

      at handler (src/handlers/addContact.ts:51:13)
      at async Object.<anonymous> (src/__tests__/handlers/addContact.test.ts:19:22)
`